src/pages/workspaces/workspace/Data.js

```diff
@@ -558,15 +559,23 @@ const DataTableActions = ({ workspace, tableName, rowCount, onRenameTable, onDel
       objectType: 'table',
       objectName: tableName,
       onDismiss: () => setDeleting(false),
-      onConfirm: _.flow(
-        Utils.withBusyState(setLoading),
-        withErrorReporting('Error deleting table')
-      )(async () => {
-        await Ajax().Workspaces.workspace(namespace, name).deleteEntitiesOfType(tableName)
-        Ajax().Metrics.captureEvent(Events.workspaceDataDeleteTable, {
-          ...extractWorkspaceDetails(workspace.workspace)
-        })
-        onDeleteTable(tableName)
+      onConfirm: Utils.withBusyState(setLoading)(async () => {
+        try {
+          await Ajax().Workspaces.workspace(namespace, name).deleteEntitiesOfType(tableName)
+          Ajax().Metrics.captureEvent(Events.workspaceDataDeleteTable, {
+            ...extractWorkspaceDetails(workspace.workspace)
+          })
+          onDeleteTable(tableName)
+        } catch (error) {
+          setDeleting(false)
+          if (error.status === 409) {
```

Comment:
I assume we get `409` only when a table can't be deleted due to would-be-dangling references, which seems to be the case based on Rawls swagger.

![image](https://user-images.githubusercontent.com/5438223/173237358-c3619d30-fc50-41f0-8921-9080d53f8a13.png)

Reply:
Yes. Deleting selected rows behaves the same way.
https://rawls.dsde-prod.broadinstitute.org/#/entities/delete_entities